auth/grants.go

```diff
@@ -146,6 +146,9 @@ type VideoGrant struct {
 
 	// if a participant can subscribe to metrics
 	CanSubscribeMetrics *bool `json:"canSubscribeMetrics,omitempty"`
+
+	// destination room which this participant can forward to
+	ForwardDestRoom string `json:"forwardDestRoom,omitempty"`
```

Comment:
same comment as in JS. I think it's better to spell out the grant, versus abbreviating, and making it less specific to forwarding.. (i.e. `DestinationRoom`)